src/top-nav-options/index.js

```diff
@@ -9,32 +9,30 @@ function TopNavBar(loggedIn) {
         navigate(`/search/${input}`);
     }
     return(
-        <nav className="navbar navbar-expand-lg navbar-dark bg-primary w-100 mb-4 ps-3 pe-3 p-sm-0">
-            <div className="container container-fluid">
-                <a className="navbar-brand" href="#">Website Title</a>
+        <nav className="navbar navbar-expand-md navbar-dark bg-primary w-100 mb-4 ps-3 pe-3 p-sm-0">
+            <div className="container-fluid m-4">
+                <a className="navbar-brand" href="#">Website name</a>
                 <button className="navbar-toggler" type="button" data-bs-toggle="collapse"
                         data-bs-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false"
                         aria-label="Toggle navigation">
-                    <span className="navbar-toggler-icon"/>
+                    <span className="navbar-toggler-icon"></span>
```

Comment:
If we don't have anything inside a tag, I think its easier to read if we use the self closing tag: "/>" at the end instead of the separate closing tag?